os/drivers/serial/Kconfig

```diff
@@ -628,7 +628,7 @@ config USART0_SERIAL_CONSOLE
 
 config UART1_SERIAL_CONSOLE
 	bool "UART1"
-	depends on ARCH_HAVE_UART1
+	depends on ARCH_HAVE_UART1 || ARCH_BOARD_SABRE_6QUAD
```

Comment:
This is not ok. Please check how it is done in other boards. 
We cannot add the board config here. Similarly check all other Kconfig files.